[2022-10-12T11:56:03.243+0000] {processor.py:156} INFO - Started process (PID=47) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T11:56:03.250+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T11:56:03.251+0000] {logging_mixin.py:117} INFO - [2022-10-12T11:56:03.251+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T11:56:04.576+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T11:56:04.674+0000] {logging_mixin.py:117} INFO - [2022-10-12T11:56:04.673+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T11:56:04.762+0000] {logging_mixin.py:117} INFO - [2022-10-12T11:56:04.761+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T11:56:04.846+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 1.611 seconds
[2022-10-12T11:56:35.603+0000] {processor.py:156} INFO - Started process (PID=104) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T11:56:35.604+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T11:56:35.605+0000] {logging_mixin.py:117} INFO - [2022-10-12T11:56:35.605+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T11:56:36.330+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T11:56:36.397+0000] {logging_mixin.py:117} INFO - [2022-10-12T11:56:36.396+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T11:56:36.467+0000] {logging_mixin.py:117} INFO - [2022-10-12T11:56:36.466+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T11:56:36.529+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.931 seconds
[2022-10-12T11:57:07.057+0000] {processor.py:156} INFO - Started process (PID=163) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T11:57:07.065+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T11:57:07.066+0000] {logging_mixin.py:117} INFO - [2022-10-12T11:57:07.065+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T11:57:07.856+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T11:57:07.923+0000] {logging_mixin.py:117} INFO - [2022-10-12T11:57:07.922+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T11:57:07.983+0000] {logging_mixin.py:117} INFO - [2022-10-12T11:57:07.982+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T11:57:08.033+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.982 seconds
[2022-10-12T11:57:38.245+0000] {processor.py:156} INFO - Started process (PID=231) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T11:57:38.246+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T11:57:38.248+0000] {logging_mixin.py:117} INFO - [2022-10-12T11:57:38.248+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T11:57:39.078+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T11:57:39.186+0000] {logging_mixin.py:117} INFO - [2022-10-12T11:57:39.185+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T11:57:39.262+0000] {logging_mixin.py:117} INFO - [2022-10-12T11:57:39.262+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T11:57:39.312+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 1.075 seconds
[2022-10-12T11:58:09.505+0000] {processor.py:156} INFO - Started process (PID=288) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T11:58:09.507+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T11:58:09.508+0000] {logging_mixin.py:117} INFO - [2022-10-12T11:58:09.508+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T11:58:10.081+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T11:58:10.167+0000] {logging_mixin.py:117} INFO - [2022-10-12T11:58:10.166+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T11:58:10.239+0000] {logging_mixin.py:117} INFO - [2022-10-12T11:58:10.239+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T11:58:10.295+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.795 seconds
[2022-10-12T11:58:40.502+0000] {processor.py:156} INFO - Started process (PID=355) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T11:58:40.504+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T11:58:40.506+0000] {logging_mixin.py:117} INFO - [2022-10-12T11:58:40.505+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T11:58:41.014+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T11:58:41.088+0000] {logging_mixin.py:117} INFO - [2022-10-12T11:58:41.087+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T11:58:41.160+0000] {logging_mixin.py:117} INFO - [2022-10-12T11:58:41.159+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T11:58:41.211+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.715 seconds
[2022-10-12T11:59:11.423+0000] {processor.py:156} INFO - Started process (PID=415) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T11:59:11.430+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T11:59:11.432+0000] {logging_mixin.py:117} INFO - [2022-10-12T11:59:11.432+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T11:59:12.031+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T11:59:12.124+0000] {logging_mixin.py:117} INFO - [2022-10-12T11:59:12.123+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T11:59:12.184+0000] {logging_mixin.py:117} INFO - [2022-10-12T11:59:12.183+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T11:59:12.226+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.810 seconds
[2022-10-12T11:59:42.771+0000] {processor.py:156} INFO - Started process (PID=476) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T11:59:42.779+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T11:59:42.781+0000] {logging_mixin.py:117} INFO - [2022-10-12T11:59:42.781+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T11:59:43.358+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T11:59:43.444+0000] {logging_mixin.py:117} INFO - [2022-10-12T11:59:43.443+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T11:59:43.510+0000] {logging_mixin.py:117} INFO - [2022-10-12T11:59:43.510+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T11:59:43.554+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.788 seconds
[2022-10-12T12:00:13.822+0000] {processor.py:156} INFO - Started process (PID=544) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:00:13.823+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T12:00:13.824+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:00:13.824+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:00:14.309+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:00:14.385+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:00:14.385+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T12:00:14.462+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:00:14.462+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T12:00:14.507+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.692 seconds
[2022-10-12T12:00:44.717+0000] {processor.py:156} INFO - Started process (PID=603) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:00:44.718+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T12:00:44.720+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:00:44.719+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:00:45.178+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:00:45.250+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:00:45.249+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T12:00:45.313+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:00:45.313+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T12:00:45.359+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.649 seconds
[2022-10-12T12:01:15.526+0000] {processor.py:156} INFO - Started process (PID=668) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:01:15.528+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T12:01:15.529+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:01:15.529+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:01:16.013+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:01:16.101+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:01:16.101+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T12:01:16.176+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:01:16.175+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T12:01:16.220+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.700 seconds
[2022-10-12T12:01:46.337+0000] {processor.py:156} INFO - Started process (PID=729) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:01:46.339+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T12:01:46.339+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:01:46.339+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:01:46.738+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:01:46.806+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:01:46.806+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T12:01:46.860+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:01:46.860+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T12:01:46.904+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.571 seconds
[2022-10-12T12:02:17.140+0000] {processor.py:156} INFO - Started process (PID=787) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:02:17.141+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T12:02:17.143+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:02:17.142+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:02:17.617+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:02:17.693+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:02:17.692+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T12:02:17.773+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:02:17.773+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T12:02:17.830+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.696 seconds
[2022-10-12T12:02:48.653+0000] {processor.py:156} INFO - Started process (PID=856) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:02:48.660+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T12:02:48.662+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:02:48.661+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:02:49.146+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:02:49.213+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:02:49.213+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T12:02:49.284+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:02:49.283+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T12:02:49.329+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.683 seconds
[2022-10-12T12:03:19.421+0000] {processor.py:156} INFO - Started process (PID=914) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:03:19.423+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T12:03:19.424+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:03:19.424+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:03:19.939+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:03:20.011+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:03:20.010+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T12:03:20.072+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:03:20.072+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T12:03:20.118+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.704 seconds
[2022-10-12T12:03:50.733+0000] {processor.py:156} INFO - Started process (PID=983) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:03:50.735+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T12:03:50.736+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:03:50.736+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:03:51.315+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:03:51.403+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:03:51.402+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T12:03:51.485+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:03:51.485+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T12:03:51.547+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.821 seconds
[2022-10-12T12:04:21.744+0000] {processor.py:156} INFO - Started process (PID=1041) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:04:21.745+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T12:04:21.746+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:04:21.746+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:04:22.199+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:04:22.278+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:04:22.278+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T12:04:22.342+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:04:22.341+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T12:04:22.386+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.649 seconds
[2022-10-12T12:04:52.444+0000] {processor.py:156} INFO - Started process (PID=1100) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:04:52.446+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T12:04:52.447+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:04:52.447+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:04:52.954+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:04:53.034+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:04:53.033+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T12:04:53.110+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:04:53.109+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T12:04:53.168+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.730 seconds
[2022-10-12T12:05:23.477+0000] {processor.py:156} INFO - Started process (PID=1171) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:05:23.479+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T12:05:23.480+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:05:23.480+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:05:24.024+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:05:24.085+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:05:24.084+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T12:05:24.150+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:05:24.149+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T12:05:24.188+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.720 seconds
[2022-10-12T12:05:54.239+0000] {processor.py:156} INFO - Started process (PID=1228) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:05:54.240+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T12:05:54.241+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:05:54.241+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:05:54.787+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:05:54.874+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:05:54.873+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T12:05:54.942+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:05:54.942+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T12:05:54.990+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.757 seconds
[2022-10-12T12:06:25.220+0000] {processor.py:156} INFO - Started process (PID=1295) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:06:25.222+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T12:06:25.223+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:06:25.223+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:06:25.709+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:06:25.793+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:06:25.792+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T12:06:25.863+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:06:25.863+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T12:06:25.905+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.691 seconds
[2022-10-12T12:06:56.268+0000] {processor.py:156} INFO - Started process (PID=1352) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:06:56.270+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T12:06:56.272+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:06:56.272+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:06:56.821+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:06:56.904+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:06:56.903+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T12:06:56.984+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:06:56.983+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T12:06:57.035+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.774 seconds
[2022-10-12T12:07:27.828+0000] {processor.py:156} INFO - Started process (PID=1410) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:07:27.829+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T12:07:27.831+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:07:27.830+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:07:28.502+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:07:28.589+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:07:28.588+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T12:07:28.666+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:07:28.666+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T12:07:28.720+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.898 seconds
[2022-10-12T12:07:59.276+0000] {processor.py:156} INFO - Started process (PID=1477) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:07:59.279+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T12:07:59.280+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:07:59.280+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:07:59.957+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:08:00.049+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:08:00.048+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T12:08:00.122+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:08:00.122+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T12:08:00.177+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.908 seconds
[2022-10-12T12:08:30.237+0000] {processor.py:156} INFO - Started process (PID=1537) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:08:30.239+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T12:08:30.240+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:08:30.240+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:08:30.830+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:08:30.909+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:08:30.908+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T12:08:30.976+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:08:30.976+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T12:08:31.032+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.802 seconds
[2022-10-12T12:09:01.381+0000] {processor.py:156} INFO - Started process (PID=1605) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:09:01.389+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T12:09:01.391+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:09:01.391+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:09:02.079+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:09:02.172+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:09:02.171+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T12:09:02.241+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:09:02.240+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T12:09:02.293+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.920 seconds
[2022-10-12T12:09:32.560+0000] {processor.py:156} INFO - Started process (PID=1665) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:09:32.562+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T12:09:32.563+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:09:32.563+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:09:33.004+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:09:33.089+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:09:33.088+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T12:09:33.162+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:09:33.162+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T12:09:33.205+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.653 seconds
[2022-10-12T12:10:03.736+0000] {processor.py:156} INFO - Started process (PID=1723) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:10:03.743+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T12:10:03.745+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:10:03.744+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:10:04.246+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:10:04.317+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:10:04.316+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T12:10:04.369+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:10:04.369+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T12:10:04.406+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.676 seconds
[2022-10-12T12:10:35.141+0000] {processor.py:156} INFO - Started process (PID=1792) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:10:35.149+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T12:10:35.150+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:10:35.149+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:10:35.612+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:10:35.682+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:10:35.681+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T12:10:35.755+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:10:35.754+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T12:10:35.801+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.665 seconds
[2022-10-12T12:11:06.101+0000] {processor.py:156} INFO - Started process (PID=1850) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:11:06.110+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T12:11:06.112+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:11:06.111+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:11:06.763+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:11:06.842+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:11:06.841+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T12:11:06.911+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:11:06.911+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T12:11:06.962+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.868 seconds
[2022-10-12T12:11:37.069+0000] {processor.py:156} INFO - Started process (PID=1909) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:11:37.077+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T12:11:37.080+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:11:37.079+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:11:38.318+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:11:38.450+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:11:38.449+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T12:11:38.670+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:11:38.669+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T12:11:38.833+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 1.783 seconds
[2022-10-12T12:12:09.000+0000] {processor.py:156} INFO - Started process (PID=1977) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:12:09.003+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T12:12:09.005+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:12:09.004+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:12:09.697+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:12:09.779+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:12:09.777+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T12:12:09.859+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:12:09.858+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T12:12:09.926+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.936 seconds
[2022-10-12T12:12:40.242+0000] {processor.py:156} INFO - Started process (PID=2034) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:12:40.243+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T12:12:40.245+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:12:40.244+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:12:40.802+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:12:40.908+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:12:40.907+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T12:12:40.986+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:12:40.985+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T12:12:41.040+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.806 seconds
[2022-10-12T12:13:11.125+0000] {processor.py:156} INFO - Started process (PID=2094) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:13:11.127+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T12:13:11.128+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:13:11.127+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:13:11.718+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:13:11.789+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:13:11.788+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T12:13:11.842+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:13:11.842+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T12:13:11.879+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.762 seconds
[2022-10-12T12:13:41.945+0000] {processor.py:156} INFO - Started process (PID=2152) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:13:41.947+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T12:13:41.949+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:13:41.948+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:13:42.473+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:13:42.554+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:13:42.553+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T12:13:42.619+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:13:42.619+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T12:13:42.678+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.742 seconds
[2022-10-12T12:14:12.811+0000] {processor.py:156} INFO - Started process (PID=2219) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:14:12.813+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T12:14:12.815+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:14:12.815+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:14:13.570+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T12:14:13.678+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:14:13.676+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T12:14:13.787+0000] {logging_mixin.py:117} INFO - [2022-10-12T12:14:13.787+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T12:14:13.899+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 1.094 seconds
[2022-10-12T20:59:26.338+0000] {processor.py:156} INFO - Started process (PID=2231) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T20:59:26.353+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T20:59:26.354+0000] {logging_mixin.py:117} INFO - [2022-10-12T20:59:26.354+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T20:59:27.091+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T20:59:27.154+0000] {logging_mixin.py:117} INFO - [2022-10-12T20:59:27.153+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T20:59:27.215+0000] {logging_mixin.py:117} INFO - [2022-10-12T20:59:27.215+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T20:59:27.253+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.920 seconds
[2022-10-12T20:59:57.472+0000] {processor.py:156} INFO - Started process (PID=2296) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T20:59:57.473+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T20:59:57.474+0000] {logging_mixin.py:117} INFO - [2022-10-12T20:59:57.474+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T20:59:58.010+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T20:59:58.081+0000] {logging_mixin.py:117} INFO - [2022-10-12T20:59:58.081+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T20:59:58.143+0000] {logging_mixin.py:117} INFO - [2022-10-12T20:59:58.142+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T20:59:58.176+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.710 seconds
[2022-10-12T21:00:28.311+0000] {processor.py:156} INFO - Started process (PID=2354) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:00:28.312+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:00:28.314+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:00:28.313+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:00:28.671+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:00:28.726+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:00:28.726+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T21:00:28.782+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:00:28.782+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T21:00:28.815+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.511 seconds
[2022-10-12T21:00:59.135+0000] {processor.py:156} INFO - Started process (PID=2421) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:00:59.135+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:00:59.136+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:00:59.136+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:00:59.457+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:00:59.509+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:00:59.509+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T21:00:59.557+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:00:59.556+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T21:00:59.585+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.454 seconds
[2022-10-12T21:01:29.778+0000] {processor.py:156} INFO - Started process (PID=2479) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:01:29.779+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:01:29.780+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:01:29.779+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:01:30.112+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:01:30.166+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:01:30.166+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T21:01:30.215+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:01:30.215+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T21:01:30.246+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.474 seconds
[2022-10-12T21:02:01.133+0000] {processor.py:156} INFO - Started process (PID=2544) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:02:01.134+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:02:01.135+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:02:01.135+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:02:01.490+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:02:01.487+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/desafioAirflow/desafio-indicium.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/desafioAirflow/desafio-indicium.py", line 4, in <module>
    from desafioAirflow.teste import leitura
ModuleNotFoundError: No module named 'desafioAirflow.teste'
[2022-10-12T21:02:01.490+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:02:01.518+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.390 seconds
[2022-10-12T21:02:31.977+0000] {processor.py:156} INFO - Started process (PID=2609) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:02:31.984+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:02:31.986+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:02:31.985+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:02:32.391+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:02:32.389+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/desafioAirflow/desafio-indicium.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/desafioAirflow/desafio-indicium.py", line 4, in <module>
    from desafioAirflow.teste import leitura
ModuleNotFoundError: No module named 'desafioAirflow.teste'
[2022-10-12T21:02:32.392+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:02:32.425+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.455 seconds
[2022-10-12T21:03:02.669+0000] {processor.py:156} INFO - Started process (PID=2665) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:03:02.670+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:03:02.671+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:03:02.671+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:03:03.050+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:03:03.048+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/desafioAirflow/desafio-indicium.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/desafioAirflow/desafio-indicium.py", line 4, in <module>
    from desafioAirflow.teste import leitura
ModuleNotFoundError: No module named 'desafioAirflow.teste'
[2022-10-12T21:03:03.051+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:03:03.084+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.421 seconds
[2022-10-12T21:03:33.429+0000] {processor.py:156} INFO - Started process (PID=2730) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:03:33.436+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:03:33.437+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:03:33.436+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:03:33.780+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:03:33.778+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/desafioAirflow/desafio-indicium.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/desafioAirflow/desafio-indicium.py", line 4, in <module>
    from desafioAirflow.teste import leitura
ModuleNotFoundError: No module named 'desafioAirflow.teste'
[2022-10-12T21:03:33.781+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:03:33.809+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.388 seconds
[2022-10-12T21:04:04.213+0000] {processor.py:156} INFO - Started process (PID=2786) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:04:04.220+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:04:04.221+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:04:04.221+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:04:04.544+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:04:04.543+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/desafioAirflow/desafio-indicium.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/desafioAirflow/desafio-indicium.py", line 4, in <module>
    from desafioAirflow.teste import leitura
ModuleNotFoundError: No module named 'desafioAirflow.teste'
[2022-10-12T21:04:04.545+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:04:04.573+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.369 seconds
[2022-10-12T21:04:34.974+0000] {processor.py:156} INFO - Started process (PID=2851) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:04:34.976+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:04:34.978+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:04:34.977+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:04:35.309+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:04:35.308+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/desafioAirflow/desafio-indicium.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/desafioAirflow/desafio-indicium.py", line 4, in <module>
    from desafioAirflow.teste import leitura
ModuleNotFoundError: No module named 'desafioAirflow.teste'
[2022-10-12T21:04:35.310+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:04:35.340+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.373 seconds
[2022-10-12T21:05:05.391+0000] {processor.py:156} INFO - Started process (PID=2916) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:05:05.398+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:05:05.399+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:05:05.398+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:05:05.725+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:05:05.723+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/desafioAirflow/desafio-indicium.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/desafioAirflow/desafio-indicium.py", line 4, in <module>
    from desafioAirflow.teste import leitura
ModuleNotFoundError: No module named 'desafioAirflow.teste'
[2022-10-12T21:05:05.726+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:05:05.752+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.366 seconds
[2022-10-12T21:05:35.863+0000] {processor.py:156} INFO - Started process (PID=2976) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:05:35.865+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:05:35.866+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:05:35.865+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:05:36.294+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:05:36.292+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/desafioAirflow/desafio-indicium.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/desafioAirflow/desafio-indicium.py", line 4, in <module>
    from desafioAirflow.teste import leitura
ModuleNotFoundError: No module named 'desafioAirflow.teste'
[2022-10-12T21:05:36.295+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:05:36.323+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.467 seconds
[2022-10-12T21:06:06.378+0000] {processor.py:156} INFO - Started process (PID=3042) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:06:06.384+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:06:06.386+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:06:06.385+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:06:06.731+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:06:06.729+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/desafioAirflow/desafio-indicium.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/desafioAirflow/desafio-indicium.py", line 4, in <module>
    from desafioAirflow.teste import leitura
ModuleNotFoundError: No module named 'desafioAirflow.teste'
[2022-10-12T21:06:06.731+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:06:06.761+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.389 seconds
[2022-10-12T21:06:36.859+0000] {processor.py:156} INFO - Started process (PID=3100) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:06:36.861+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:06:36.862+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:06:36.862+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:06:37.181+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:06:37.179+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/desafioAirflow/desafio-indicium.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/desafioAirflow/desafio-indicium.py", line 4, in <module>
    from desafioAirflow.teste import leitura
ModuleNotFoundError: No module named 'desafioAirflow.teste'
[2022-10-12T21:06:37.181+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:06:37.206+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.352 seconds
[2022-10-12T21:07:07.348+0000] {processor.py:156} INFO - Started process (PID=3168) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:07:07.349+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:07:07.350+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:07:07.349+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:07:07.647+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:07:07.645+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/desafioAirflow/desafio-indicium.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/desafioAirflow/desafio-indicium.py", line 4, in <module>
    from desafioAirflow.teste import leitura
ModuleNotFoundError: No module named 'desafioAirflow.teste'
[2022-10-12T21:07:07.647+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:07:07.668+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.325 seconds
[2022-10-12T21:07:37.854+0000] {processor.py:156} INFO - Started process (PID=3226) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:07:37.855+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:07:37.856+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:07:37.856+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:07:38.176+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:07:38.174+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/desafioAirflow/desafio-indicium.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/desafioAirflow/desafio-indicium.py", line 4, in <module>
    from desafioAirflow.teste import leitura
ModuleNotFoundError: No module named 'desafioAirflow.teste'
[2022-10-12T21:07:38.176+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:07:38.204+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.357 seconds
[2022-10-12T21:08:08.683+0000] {processor.py:156} INFO - Started process (PID=3293) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:08:08.685+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:08:08.686+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:08:08.685+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:08:08.982+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:08:08.981+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/desafioAirflow/desafio-indicium.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/desafioAirflow/desafio-indicium.py", line 4, in <module>
    from desafioAirflow.teste import leitura
ModuleNotFoundError: No module named 'desafioAirflow.teste'
[2022-10-12T21:08:08.983+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:08:09.011+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.333 seconds
[2022-10-12T21:08:39.081+0000] {processor.py:156} INFO - Started process (PID=3360) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:08:39.088+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:08:39.090+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:08:39.089+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:08:39.466+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:08:39.463+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/desafioAirflow/desafio-indicium.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/desafioAirflow/desafio-indicium.py", line 4, in <module>
    from desafioAirflow.teste import leitura
ModuleNotFoundError: No module named 'desafioAirflow.teste'
[2022-10-12T21:08:39.467+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:08:39.506+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.431 seconds
[2022-10-12T21:09:09.606+0000] {processor.py:156} INFO - Started process (PID=3418) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:09:09.613+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:09:09.614+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:09:09.614+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:09:10.051+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:09:10.049+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/desafioAirflow/desafio-indicium.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/desafioAirflow/desafio-indicium.py", line 4, in <module>
    from desafioAirflow.teste import leitura
ModuleNotFoundError: No module named 'desafioAirflow.teste'
[2022-10-12T21:09:10.052+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:09:10.102+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.503 seconds
[2022-10-12T21:09:40.556+0000] {processor.py:156} INFO - Started process (PID=3486) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:09:40.558+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:09:40.558+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:09:40.558+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:09:40.848+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:09:40.846+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/desafioAirflow/desafio-indicium.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/desafioAirflow/desafio-indicium.py", line 4, in <module>
    from desafioAirflow.teste import leitura
ModuleNotFoundError: No module named 'desafioAirflow.teste'
[2022-10-12T21:09:40.848+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:09:40.875+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.326 seconds
[2022-10-12T21:10:11.377+0000] {processor.py:156} INFO - Started process (PID=3544) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:10:11.384+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:10:11.386+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:10:11.386+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:10:11.790+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:10:11.788+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/desafioAirflow/desafio-indicium.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/desafioAirflow/desafio-indicium.py", line 4, in <module>
    from desafioAirflow.teste import leitura
ModuleNotFoundError: No module named 'desafioAirflow.teste'
[2022-10-12T21:10:11.791+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:10:11.824+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.455 seconds
[2022-10-12T21:10:41.938+0000] {processor.py:156} INFO - Started process (PID=3612) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:10:41.945+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:10:41.947+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:10:41.946+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:10:42.273+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:10:42.271+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/desafioAirflow/desafio-indicium.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/desafioAirflow/desafio-indicium.py", line 4, in <module>
    from desafioAirflow.teste import leitura
ModuleNotFoundError: No module named 'desafioAirflow.teste'
[2022-10-12T21:10:42.273+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:10:42.300+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.368 seconds
[2022-10-12T21:11:12.392+0000] {processor.py:156} INFO - Started process (PID=3680) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:11:12.393+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:11:12.395+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:11:12.395+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:11:12.788+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:11:12.787+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/desafioAirflow/desafio-indicium.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/desafioAirflow/desafio-indicium.py", line 4, in <module>
    from desafioAirflow.teste import leitura
ModuleNotFoundError: No module named 'desafioAirflow.teste'
[2022-10-12T21:11:12.789+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:11:12.813+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.428 seconds
[2022-10-12T21:11:43.681+0000] {processor.py:156} INFO - Started process (PID=3740) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:11:43.688+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:11:43.690+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:11:43.690+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:11:44.089+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:11:44.087+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/desafioAirflow/desafio-indicium.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/desafioAirflow/desafio-indicium.py", line 4, in <module>
    from desafioAirflow.teste import leitura
ModuleNotFoundError: No module named 'desafioAirflow.teste'
[2022-10-12T21:11:44.090+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:11:44.121+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.446 seconds
[2022-10-12T21:12:14.335+0000] {processor.py:156} INFO - Started process (PID=3807) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:12:14.336+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:12:14.337+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:12:14.337+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:12:14.663+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:12:14.661+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/desafioAirflow/desafio-indicium.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/desafioAirflow/desafio-indicium.py", line 4, in <module>
    from desafioAirflow.teste import leitura
ModuleNotFoundError: No module named 'desafioAirflow.teste'
[2022-10-12T21:12:14.663+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:12:14.705+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.375 seconds
[2022-10-12T21:12:44.946+0000] {processor.py:156} INFO - Started process (PID=3865) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:12:44.953+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:12:44.954+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:12:44.954+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:12:45.281+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:12:45.278+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/desafioAirflow/desafio-indicium.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/desafioAirflow/desafio-indicium.py", line 4, in <module>
    from desafioAirflow.teste import leitura
ModuleNotFoundError: No module named 'desafioAirflow.teste'
[2022-10-12T21:12:45.282+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:12:45.305+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.365 seconds
[2022-10-12T21:13:15.537+0000] {processor.py:156} INFO - Started process (PID=3931) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:13:15.538+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:13:15.539+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:13:15.539+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:13:15.860+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:13:15.858+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/desafioAirflow/desafio-indicium.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/desafioAirflow/desafio-indicium.py", line 4, in <module>
    from desafioAirflow.teste import leitura
ModuleNotFoundError: No module named 'desafioAirflow.teste'
[2022-10-12T21:13:15.860+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:13:15.890+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.358 seconds
[2022-10-12T21:13:46.043+0000] {processor.py:156} INFO - Started process (PID=4003) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:13:46.045+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:13:46.045+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:13:46.045+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:13:46.363+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:13:46.361+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/desafioAirflow/desafio-indicium.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/desafioAirflow/desafio-indicium.py", line 4, in <module>
    from desafioAirflow.teste import leitura
ModuleNotFoundError: No module named 'desafioAirflow.teste'
[2022-10-12T21:13:46.364+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:13:46.390+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.352 seconds
[2022-10-12T21:14:16.448+0000] {processor.py:156} INFO - Started process (PID=4059) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:14:16.449+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:14:16.450+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:14:16.450+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:14:16.992+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:14:16.989+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/desafioAirflow/desafio-indicium.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/desafioAirflow/desafio-indicium.py", line 4, in <module>
    from desafioAirflow.teste import leitura
ModuleNotFoundError: No module named 'desafioAirflow.teste'
[2022-10-12T21:14:16.993+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:14:17.052+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.609 seconds
[2022-10-12T21:14:47.701+0000] {processor.py:156} INFO - Started process (PID=4126) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:14:47.702+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:14:47.704+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:14:47.703+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:14:48.074+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:14:48.073+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/desafioAirflow/desafio-indicium.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/desafioAirflow/desafio-indicium.py", line 4, in <module>
    from desafioAirflow.teste import leitura
ModuleNotFoundError: No module named 'desafioAirflow.teste'
[2022-10-12T21:14:48.075+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:14:48.100+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.406 seconds
[2022-10-12T21:15:18.900+0000] {processor.py:156} INFO - Started process (PID=4184) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:15:18.908+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:15:18.909+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:15:18.909+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:15:19.254+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:15:19.253+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/desafioAirflow/desafio-indicium.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/desafioAirflow/desafio-indicium.py", line 4, in <module>
    from desafioAirflow.teste import leitura
ModuleNotFoundError: No module named 'desafioAirflow.teste'
[2022-10-12T21:15:19.255+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:15:19.282+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.387 seconds
[2022-10-12T21:15:49.359+0000] {processor.py:156} INFO - Started process (PID=4249) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:15:49.367+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:15:49.368+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:15:49.368+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:15:49.744+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:15:49.741+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/desafioAirflow/desafio-indicium.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/desafioAirflow/desafio-indicium.py", line 4, in <module>
    from desafioAirflow.teste import leitura
ModuleNotFoundError: No module named 'desafioAirflow.teste'
[2022-10-12T21:15:49.744+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:15:49.788+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.435 seconds
[2022-10-12T21:16:20.046+0000] {processor.py:156} INFO - Started process (PID=4316) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:16:20.053+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:16:20.054+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:16:20.053+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:16:20.411+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:16:20.410+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/desafioAirflow/desafio-indicium.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/desafioAirflow/desafio-indicium.py", line 4, in <module>
    from desafioAirflow.teste import leitura
ModuleNotFoundError: No module named 'desafioAirflow.teste'
[2022-10-12T21:16:20.412+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:16:20.443+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.403 seconds
[2022-10-12T21:16:28.010+0000] {processor.py:156} INFO - Started process (PID=4352) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:16:28.018+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:16:28.020+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:16:28.019+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:16:28.395+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:16:28.543+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:16:28.543+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T21:16:28.602+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:16:28.602+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T21:16:28.653+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.651 seconds
[2022-10-12T21:16:59.487+0000] {processor.py:156} INFO - Started process (PID=4422) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:16:59.494+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:16:59.495+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:16:59.495+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:16:59.863+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:16:59.927+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:16:59.927+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T21:16:59.980+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:16:59.980+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T21:17:00.011+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.529 seconds
[2022-10-12T21:17:30.601+0000] {processor.py:156} INFO - Started process (PID=4490) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:17:30.608+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:17:30.610+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:17:30.609+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:17:30.968+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:17:31.030+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:17:31.030+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T21:17:31.084+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:17:31.084+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T21:17:31.125+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.529 seconds
[2022-10-12T21:17:54.522+0000] {processor.py:156} INFO - Started process (PID=4528) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:17:54.523+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:17:54.527+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:17:54.527+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:17:54.904+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:17:55.037+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:17:55.036+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T21:17:55.085+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:17:55.085+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T21:17:55.120+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.605 seconds
[2022-10-12T21:18:25.160+0000] {processor.py:156} INFO - Started process (PID=4594) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:18:25.162+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:18:25.163+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:18:25.162+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:18:25.524+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:18:25.578+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:18:25.578+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T21:18:25.616+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:18:25.616+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T21:18:25.645+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.491 seconds
[2022-10-12T21:18:55.738+0000] {processor.py:156} INFO - Started process (PID=4660) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:18:55.739+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:18:55.739+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:18:55.739+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:18:56.076+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:18:56.133+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:18:56.133+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T21:18:56.177+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:18:56.177+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T21:18:56.224+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.492 seconds
[2022-10-12T21:19:26.829+0000] {processor.py:156} INFO - Started process (PID=4718) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:19:26.836+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:19:26.836+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:19:26.836+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:19:27.178+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:19:27.224+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:19:27.224+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T21:19:27.267+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:19:27.267+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T21:19:27.309+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.486 seconds
[2022-10-12T21:19:57.776+0000] {processor.py:156} INFO - Started process (PID=4785) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:19:57.778+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:19:57.779+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:19:57.779+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:19:58.130+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:19:58.189+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:19:58.189+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T21:19:58.236+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:19:58.236+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T21:19:58.264+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.491 seconds
[2022-10-12T21:20:28.851+0000] {processor.py:156} INFO - Started process (PID=4842) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:20:28.852+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:20:28.853+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:20:28.853+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:20:29.249+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:20:29.306+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:20:29.306+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T21:20:29.352+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:20:29.352+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T21:20:29.391+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.544 seconds
[2022-10-12T21:20:59.647+0000] {processor.py:156} INFO - Started process (PID=4909) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:20:59.649+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:20:59.650+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:20:59.650+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:20:59.967+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:21:00.016+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:21:00.016+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T21:21:00.061+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:21:00.061+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T21:21:00.096+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.453 seconds
[2022-10-12T21:21:30.138+0000] {processor.py:156} INFO - Started process (PID=4967) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:21:30.139+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:21:30.140+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:21:30.140+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:21:30.510+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:21:30.578+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:21:30.577+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T21:21:30.642+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:21:30.642+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T21:21:30.687+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.553 seconds
[2022-10-12T21:22:00.743+0000] {processor.py:156} INFO - Started process (PID=5035) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:22:00.744+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:22:00.745+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:22:00.745+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:22:01.055+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:22:01.103+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:22:01.103+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T21:22:01.141+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:22:01.141+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T21:22:01.167+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.429 seconds
[2022-10-12T21:22:31.879+0000] {processor.py:156} INFO - Started process (PID=5099) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:22:31.880+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:22:31.881+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:22:31.881+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:22:32.290+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:22:32.360+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:22:32.360+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T21:22:32.401+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:22:32.401+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T21:22:32.429+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.553 seconds
[2022-10-12T21:23:02.838+0000] {processor.py:156} INFO - Started process (PID=5160) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:23:02.839+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:23:02.840+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:23:02.840+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:23:03.176+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:23:03.233+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:23:03.232+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T21:23:03.283+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:23:03.283+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T21:23:03.314+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.482 seconds
[2022-10-12T21:23:33.379+0000] {processor.py:156} INFO - Started process (PID=5228) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:23:33.380+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:23:33.381+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:23:33.381+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:23:33.734+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:23:33.796+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:23:33.795+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T21:23:33.850+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:23:33.850+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T21:23:33.879+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.502 seconds
[2022-10-12T21:24:03.962+0000] {processor.py:156} INFO - Started process (PID=5284) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:24:03.962+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:24:03.963+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:24:03.963+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:24:04.285+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:24:04.332+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:24:04.332+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T21:24:04.380+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:24:04.380+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T21:24:04.416+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.459 seconds
[2022-10-12T21:24:34.621+0000] {processor.py:156} INFO - Started process (PID=5352) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:24:34.622+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:24:34.623+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:24:34.623+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:24:35.055+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:24:35.125+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:24:35.124+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T21:24:35.188+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:24:35.188+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T21:24:35.221+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.607 seconds
[2022-10-12T21:25:05.550+0000] {processor.py:156} INFO - Started process (PID=5412) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:25:05.551+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:25:05.553+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:25:05.552+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:25:05.987+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:25:06.047+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:25:06.046+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T21:25:06.087+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:25:06.087+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T21:25:06.118+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.574 seconds
[2022-10-12T21:25:36.332+0000] {processor.py:156} INFO - Started process (PID=5483) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:25:36.333+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:25:36.334+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:25:36.334+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:25:36.724+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:25:36.774+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:25:36.774+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T21:25:36.822+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:25:36.821+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T21:25:36.855+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.530 seconds
[2022-10-12T21:26:07.826+0000] {processor.py:156} INFO - Started process (PID=5551) to work on /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:26:07.827+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/desafioAirflow/desafio-indicium.py for tasks to queue
[2022-10-12T21:26:07.828+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:26:07.828+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:26:08.239+0000] {processor.py:768} INFO - DAG(s) dict_keys(['DesafioAirflow']) retrieved from /opt/airflow/dags/desafioAirflow/desafio-indicium.py
[2022-10-12T21:26:08.301+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:26:08.301+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-12T21:26:08.347+0000] {logging_mixin.py:117} INFO - [2022-10-12T21:26:08.347+0000] {dag.py:3328} INFO - Setting next_dagrun for DesafioAirflow to 2022-10-12T00:00:00+00:00, run_after=2022-10-13T00:00:00+00:00
[2022-10-12T21:26:08.374+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/desafioAirflow/desafio-indicium.py took 0.556 seconds
